.circleci/config.yml

```diff
@@ -310,6 +310,18 @@ jobs:
       - run: pip install wheel twine
       - run: python -m twine upload --non-interactive --verbose --repository pypi dist/*
 
+  publish-spec:
+    working_directory: ~/openlineage
+    docker:
+      - image: cimg/base:2021.07
+    steps:
+      - *checkout_project_root
+      - add_ssh_keys:
+          fingerprints:
+            - "SO:ME:FIN:G:ER:PR:IN:T"
```

Comment:
is that were we need to add the key for a ci user?

Reply:
Added user https://github.com/openlineage-github-bot